.github/workflows/ci.yml

```diff
@@ -136,4 +139,53 @@ jobs:
           source cacitesting.env
           python -m unittest discover -s tests/attestation
 
+  test-skr:
```

Comment:
We should add a cleanup job to this file so that things get cleaned up after the run

Reply:
The c_aci_testing package handles cleanup for us, when `target_run_ctx` exits it cleans up deployed resources

Reply:
We should probably add some info on this in the README or even a new README in the workflows dir